tests/test_xet_download.py

```diff
@@ -0,0 +1,77 @@
+import os
+from unittest.mock import patch
+
+import pytest
+
+from huggingface_hub.file_download import get_hf_file_metadata, hf_hub_download, hf_hub_url, try_to_load_from_cache
+
+from .testing_utils import (
+    DUMMY_XET_MODEL_ID,
+    requires,
+    with_production_testing,
+)
+
+
+@requires("hf_xet")
+@with_production_testing
+class TestXetDownload:
+    file_name = "dummy.txt"
+    xet_file_name = "dummy.safetensors"
```

Comment:
since `DUMMY_XET_MODEL_ID` is a constant, I'd make them some constants as well (`DUMMY_XET_FILE` / `DUMMY_XET_REGULAR_FILE` ?)